examples/minimal/undefined/UB-CB1-bad.output

```diff
@@ -0,0 +1,8 @@
+Error: CB1
+Description: types of function call arguments aren't compatible with declared types after promotions.
+Type: Undefined behavior.
+See also: C11 sec. 6.5.2.2:6, J.2:1 item 39
+  at f(/Users/manasvi/Documents/examples/allbad/j040a.c:7)
```

Comment:
you need to run these from the same directory the files exist in so that they don't get prefixed with the absolute path which is dependent on the machine. Take a look at what they look like in the tests/unit-fail directory.